src/backend/modules/company/company.service.ts

```diff
@@ -10,6 +10,12 @@ export class CompanyService {
   async getMany(query: GetCompaniesQuery) {
     const take = query.take ? query.take : 10;
 
+    let orderBy: string = "";
+    if(query.orderBy === "quantity" || query.orderBy ===  "priceCents"){
+      orderBy = query.orderBy;
+      query.orderBy = undefined;
+    }
+
```

Comment:
Prisma doesn't have deep orderBy unfortunately so I understand why it's there.
But to have fully working sort you also need to make custom pagination not rely on prisma.
Because later sorting in the function will only sort 10 elements, not all

So I see two ways of doing it:
- remove "skip" and "take" from request and create pagination later (similar to sorting)
- write custom SQL query

Way 1 is preferred I would say.